crawl-ref/source/dat/database/de/miscname.txt

```diff
@@ -124,7 +124,7 @@ Ein schrecklicher Fluch hat dich auf die Suche nach der Kugel von Zot geschickt.
 
 Während du auf der Suche nach der Kugel hinabsteigst, scharen sich @_halloween_things_@ um dich. An diesem Ort spukt es!
 
-Du hörst das Gackern einer Hexe und bist plötzlich ins Dungeon angekommen. Wenn du die Kugel von Zot nicht findest, verwandelst du dich in @_a_lowly_@!
+Du hörst das Gackern einer Hexe und bist plötzlich ins Kerker angekommen. Wenn du die Kugel von Zot nicht findest, verwandelst du dich in @_a_lowly_@!
```

Comment:
Du hörst das Gackern einer Hexe und bist plötzlich im Kerker. Wenn du die Kugel von Zot nicht findest, verwandelst du dich in @_a_lowly_@!